README.md

```diff
@@ -41,39 +39,39 @@ In a single-cluster setup, developers typically access the cluster and deploy Ku
 
 KubeStellar simplifies this process by allowing developers to define a binding policy between clusters and Kubernetes objects. It then uses your regular single-cluster tooling to deploy and configure each cluster based on these binding policies, making multi-cluster operations as straightforward as managing a single cluster. This approach enhances productivity and efficiency, making KubeStellar a valuable tool in a multi-cluster Kubernetes environment.
 
-## Website
+## 🌐 Website
```

Comment:
Please do not add emoji into section headings.
Our style is to not have emoji in section headings. This is in part because we anticipate feeding those headings into tools that do not cope with emoji.
@KPRoche will be documenting this style point and removing the existing violation in the two copes of "readme".